cdap-docs/hydrator-manual/source/glossary.rst

```diff
@@ -95,6 +95,10 @@ Cask Hydrator Glossary
       A type of :term:`Hydrator Pipeline` that runs continuously, performing actions on
       a distinct set of data.
 
+   Secure Key
+      A key for an entry in :ref:`Secure Storage <admin-secure-storage>`. An entry in secure
+      storage can be referenced and retrieved using a Secure Key.
+
```

Comment:
This is not the correct file for this entry. It should instead be added to the main glossary, located in `reference-manual/source/glossary.rst`